### PR TITLE
Add Storybook Story to Label Component

### PR DESCRIPTION
**Pull request by natashaU**
- This PR adds a [storybook story](https://github.com/jonspalmer/view_component_storybook) to the Label Component:

Screen Shot 2020-09-17 at 8 13 05 PM

- I also added `bundle install;` to the procfile. Not sure we want to include that in this PR ❓ ❓ , but it seemed to solve gem dependency issues whenever I switched branches and needed to start the server. 

I had the same problems as described in this Issue in regards to the `type` discrepancy: https://github.com/primer/view_components/issues/82

cc/ @joelhawksley @aellispierce @jonspalmer

### CODE REVIEW

**Comment by manuelpuyol** [comment]
somehow coverage was affected 🤔

File: Procfile [hunk 1]
```diff
@@ -1,3 +1,3 @@
-stories: cd demo; bin/rails view_component_storybook:write_stories_json
-rails: cd demo; STORYBOOK=true bin/rails s -p 4000
+stories: cd demo; bundle install; bin/rails view_component_storybook:write_stories_json
```
Comment (joelhawksley) [inline Procfile:1]: @manuelpuyol any idea how to avoid needing this `bundle install`?

Comment (manuelpuyol) [reply Procfile:1]: I'm not sure, I haven't run into the issues described tbh 🤔

**Comment by natashaU** [comment]
@manuelpuyol I removed the stories helper array as per your recent PR update: https://github.com/primer/view_components/pull/91
@joelhawksley I removed `.to_sym` as per your recent PR with the gem upgrade: https://github.com/primer/view_components/pull/89

Everything looks good ~~

File: stories/primer/label_component_stories.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class Primer::LabelComponentStories < ViewComponent::Storybook::Stories
+  layout "storybook_preview"
+
+  story(:label) do
+    controls do
+      title "this is a label"
+      select(:scheme, (Primer::LabelComponent::SCHEME_MAPPINGS.keys), :blue)
+      select(:variant, (Primer::LabelComponent::VARIANT_MAPPINGS.keys), :large)
```
Comment (manuelpuyol) [inline stories/primer/label_component_stories.rb:10]:
```suggestion
      select(:scheme, Primer::LabelComponent::SCHEME_MAPPINGS.keys, :blue)
      select(:variant, Primer::LabelComponent::VARIANT_MAPPINGS.keys, :large)
```

**Review by manuelpuyol — APPROVED** [review]

**Pull request merged**